tensorflow/core/kernels/data/name_utils.cc

```diff
@@ -21,45 +21,27 @@ namespace data {
 namespace name_utils {
 
 ABSL_CONST_INIT const char kDelimiter[] = "::";
+ABSL_CONST_INIT const char kDefaultDatasetDebugStringPrefix[] = "";
+
+constexpr char kDefaultDatasetName[] = "Dataset";
```

Comment:
call it simply `kDataset` because it is used in places where it is technically not an overridable default